src/pages/evm/staking/StakingPageHeader.vue

```diff
@@ -1,107 +1,229 @@
+    isPrimaryLoading: isUnstakingLoading.value,
     lowContrastSecondaryText: true,
     useSmallBox: true,
 }, {
     label: $t('evm_stake.withdrawable_card_label'),
-    tooltip: $t('evm_stake.withdrawable_card_tooltip', { stakedSymbol: stakedTokenSymbol.value, systemSymbol: systemTokenSymbol.value, unlockPeriod: unlockPeriod.value }),
-    primaryText: withdrawableFiatValueText.value,
-    secondaryText: prettyPrintToken(withdrawableBalanceBn.value, systemTokenSymbol.value),
+    tooltip: $t('evm_stake.withdrawable_card_tooltip', { stakedSymbol: stakedToken.symbol, systemSymbol: systemToken.symbol, unlockPeriod: unlockPeriod.value }),
+    primaryText: prettyPrintToken(withdrawableFiatValueBn.value, fiatCurrency.value),
+    secondaryText: isWithdrawableLoading.value ? '' : prettyPrintToken(withdrawableBalanceBn.value, systemToken.symbol),
     lowContrastSecondaryText: true,
+    isPrimaryLoading: isWithdrawableLoading.value,
     useSmallBox: true,
 }]);
 
 const secondLineData = computed(() => [{
-    label: $t('evm_stake.apy_card_label', { symbol: systemTokenSymbol.value }),
-    tooltip: $t('evm_stake.apy_card_tooltip', { stakedSymbol: stakedTokenSymbol.value, systemSymbol: systemTokenSymbol.value }),
+    label: $t('evm_stake.apy_card_label', { symbol: systemToken.symbol }),
+    tooltip: $t('evm_stake.apy_card_tooltip', { stakedSymbol: stakedToken.symbol, systemSymbol: systemToken.symbol }),
     secondaryText: apyPrittyPrint.value,
     lowContrastSecondaryText: false,
+    isSecondaryLoading: apyisLoading.value,
     useSmallBox: true,
 }, {
     label: $t('evm_stake.unstaking_period_card_label'),
-    tooltip: $t('evm_stake.unstaking_period_card_tooltip', { stakedSymbol: stakedTokenSymbol.value, systemSymbol: systemTokenSymbol.value }),
+    tooltip: $t('evm_stake.unstaking_period_card_tooltip', { stakedSymbol: stakedToken.symbol, systemSymbol: systemToken.symbol }),
     secondaryText: unlockPeriod.value,
     lowContrastSecondaryText: false,
     useSmallBox: true,
 }, {
     label: $t('evm_stake.tvl_card_label'),
-    tooltip: $t('evm_stake.tvl_card_tooltip', { evmNetworkName: evmNetworkName.value, stakedSymbol: stakedTokenSymbol.value, systemSymbol: systemTokenSymbol.value }),
-    secondaryText: prettyPrintToken(tvlAmountBn.value, systemTokenSymbol.value),
+    tooltip: $t('evm_stake.tvl_card_tooltip', { evmNetworkName: evmNetworkName.value, stakedSymbol: stakedToken.symbol, systemSymbol: systemToken.symbol }),
+    secondaryText: prettyPrintToken(tvlAmountBn.value, systemToken.symbol),
     lowContrastSecondaryText: false,
     useSmallBox: true,
 }]);
 
+// creatmos un interval que cada 5 segundos verifique si cada uno de los loadings es true o false.
+// por cada loading que encontremos en true, debemos llamar a actualizar las dependendicas de ese loading
+// si todos los loadings son false, entonces podemos parar el intervalo
```

Comment:
¿podría traducir al inglés? gracias.

Reply:
That also was supposed to be removed.
fixed!